Jenkinsfile

```diff
@@ -22,13 +22,19 @@ pipeline {
             }
             steps {
                 script {
+                    def gitconfig = scm.userRemoteConfigs.getAt(0)
+                    git(branch: 'main',credentialsId: gitconfig.getCredentialsId(), url: 'https://github.com/Vandebron/mpyl_config.git')
```

Comment:
Do we still want to discuss alternatives later? I agree with doing this for now since there is some pressure from Arno for a quick solution

Reply:
Sure, we can always discuss alternatives.